src/main/resources/com/google/api/codegen/java/message.snip

```diff
@@ -112,7 +112,7 @@
   }
   private static final {@schema.innerTypeName} DEFAULT_INSTANCE;
   static {
-    DEFAULT_INSTANCE = new {@schema.innerTypeName}();
+      DEFAULT_INSTANCE = new {@schema.innerTypeName}();
```

Comment:
Oh, the rest of them look fine; this particular one was already correct, but it got indented too far.